### ISSUE TITLE
Button names not submitted with form after form is swapped

### ISSUE THREAD

**Issue by ekwoka** [bigskysoftware/htmx#1819]
Example:

```html
<body>
    <form
      id="hello"
      hx-target="#hello"
      hx-select="#hello"
      hx-swap="outerHTML"
      hx-get="/">
      <input type="text" name="name" />
      <button name="value" type="submit">Submit</button>
    </form>
    <button name="outside" form="hello" type="submit">Outside</button>
  </body>
  ```
  
  When submitting a form with a button external to the form, only the first use of the external button sends the buttons name with it. 
  
  Similarly, if anything else triggers htmx, it will similarly break (like a submit inside the form or some other event being listened on).

**Comment by Telroshan** [comment]
Hey, I'm not sure I understand the issue
On [this JSFiddle](https://jsfiddle.net/gusr4wtq/), every time I click the outside button, its name correctly appears in the request payload. Am I misunderstanding something?

**Comment by ekwoka** [comment]
Yes, that's missing the fact that the form isn't being swapped out.

It's just 404ing.

The issue is that the FORM is replaced (becomes a new element) but the outside BUTTON is the same as before. So it's reference to its FORM is scoped to the OLD form before the new one replaced it.

That JSfiddle never actually swaps out the form.

**Comment by Telroshan** [comment]
Oh right, I had missed that hx-target value... thanks for the details, it's indeed a bug

**Comment by BubbaJoe** [comment]
looks like this was fixes i guess, we can close this?

**Comment by ekwoka** [comment]
Yes fixed with #1820
